jupyterhub/scopes.py

```diff
@@ -23,92 +23,89 @@
 from . import roles
 
 scope_definitions = {
-    '(no_scope)': {'description': 'Allows for only identifying the owning entity.'},
+    '(no_scope)': {'description': 'Identify the owner of this entity.'},
     'self': {
-        'description': 'Metascope, grants access to user’s own resources only; resolves to (no_scope) for services.'
+        'description': 'The user’s own resources.',
+        'metadescription': 'metascope for users, resolves to (no_scope) for services',
     },
     'all': {
-        'description': 'Metascope, valid for tokens only. Grants access to everything that the token-owning entity can access.'
+        'description': 'Everything that the token-owning entity can access.',
```

Comment:
For the authorization page, I think it would really be best to have the text "you" instead of "the token-owning entity". I think the ideal user text here is probably something like "The application will be able to do anything and everything you can do." There are some tricky differences between the authorization page where you're addressing the prospective token owner and the docs, where you are talking in general terms.

What do you think?

Reply:
Agreed. I'll refactor and use a separate description for the docs, if the auth-page one does not suffice